verifyml/model_tests/FEAT/MinMaxMetricThreshold.py

```diff
@@ -141,98 +184,124 @@ def get_result(self, df_test_with_output: pd.DataFrame) -> pd.DataFrame:
           save_plots: If True, saves the plots to the class instance.
         """
         if self.result is None:
             raise AttributeError("Cannot plot before obtaining results.")
 
-        plt.figure(figsize=(15, 8))
-        colors = [
-            "red",
-            "blue",
-            "grey",
-            "green",
-            "black",
-            "brown",
-            "purple",
-            "orange",
-            "magenta",
-            "pink",
-        ]
-
-        for value in self.fpr:
-            color = colors.pop(0)
-            fpr = self.fpr[value]
-            tpr = self.tpr[value]
-            idx = self.idx[value]
-            thresholds_lst = self.thresholds_lst[value]
-            _threshold = self.thresholds[value]
-
-            optimal_idx = np.argmax(tpr - fpr)
-            optimal_threshold = thresholds_lst[optimal_idx + 1]
-            optimal_txt = "Optimal Prob Threshold"
-            txt = "Current Prob Threshold"
-
-            plt.scatter(
-                fpr[optimal_idx],
-                tpr[optimal_idx],
-                color=color,
-                marker=".",
-                s=90,
-                label=f"{optimal_txt} = {str(round(optimal_threshold,3))}, {self.attr}_{value}",
-            )
-
-            plt.scatter(
-                fpr[idx],
-                tpr[idx],
-                color=color,
-                marker="x",
-                s=50,
-                label=f"{txt} = {str(_threshold)}, {self.attr}_{value}",
-            )
-
-            plt.plot(fpr, tpr, label=f"ROC of {self.attr}_{value}", color=color)
-
-        plt.xlabel("False Positive Rate", fontsize=15)
-        plt.ylabel("True Positive Rate", fontsize=15)
-
-        if self.metric == "tpr":
-            plt.axhline(
-                y=self.threshold,
-                color="black",
-                linestyle="--",
-                label=f"Mininum TPR Threshold = {str(self.threshold)}",
-            )
-        elif self.metric == "fnr":
-            plt.axhline(
-                y=1 - self.threshold,
-                color="black",
-                linestyle="--",
-                label=f"Maximum FNR (1-TPR) Threshold = {str(self.threshold)}",
-            )
-        elif self.metric == "fpr":
-            plt.axvline(
-                x=self.threshold,
-                color="black",
-                linestyle="--",
-                label=f"Maximum FPR Threshold = {str(self.threshold)}",
-            )
-        elif self.metric == "tnr":
-            plt.axvline(
-                x=1 - self.threshold,
-                color="black",
-                linestyle="--",
-                label=f"Mininum TNR (1-FPR) Threshold = {str(self.threshold)}",
-            )
-
-        title = f"ROC Curve of {self.attr} groups"
-        plt.title(title, fontsize=15)
-        plt.legend(bbox_to_anchor=(1.05, 1), loc="upper left")
+        lower_list = []
+        upper_list = []
+        if self.metric in ["mse"]:
```

Comment:
`if self.metric == "mse"`

Reply:
i'm keeping it in a list format for now, might add other regression metrics in the future